packages/components/file-card/src/file-card.js

```diff
@@ -9,14 +9,23 @@ import '@tradeshift/elements.card';
 import '@tradeshift/elements.progress-bar';
 import '@tradeshift/elements.file-size';
 import css from './file-card.css';
+import {
+	Messages,
```

Comment:
Why this one is uppercase, others are lowercase?

Reply:
a way to demonstrate that this one is constant! but maybe is not the best way since you had to ask. What do you think? Just use lowercase like others or all in uppercase?